com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/CompletionProposalRequestor.java

```diff
@@ -159,13 +160,26 @@ public CompletionItem toCompletionItem(CompletionProposal proposal, int index) {
         data.put(CompletionResolveHandler.DATA_FIELD_PROPOSAL_ID, String.valueOf(index));
```

Comment:
Just asking, do we also have resolve support for the completion in debug console?

Reply:
No lazy-resolve support in debug console. It just inserts the insertText by default.